systemd/pkg.yaml

```diff
@@ -0,0 +1,120 @@
+        patch -p1 < /pkg/patches/0023-include-signal.h-to-avoid-the-undeclared-error.patch
+        patch -p1 < /pkg/patches/0024-undef-stdin-for-references-using-stdin-as-a-struct-m.patch
+        patch -p1 < /pkg/patches/0025-adjust-header-inclusion-order-to-avoid-redeclaration.patch
+        patch -p1 < /pkg/patches/0026-build-path.c-avoid-boot-time-segfault-for-musl.patch
+
+        # General improvement from OpenEmbedded
+        patch -p1 < /pkg/patches/0001-Do-not-create-var-log-README.patch
+
+        # Chimera Linux patch for ARM64 VMs, BSD-2-Clause licensed
+        # https://raw.githubusercontent.com/chimera-linux/cports/dabadf241e1078f5a696d6615622f67f0b1c85ac/main/udev/patches/xxx-udev-dynbuf.patch
+        patch -p1 < /pkg/patches/xxx-udev-dynbuf.patch
+
+        # Own fixes for musl
+        patch -p1 < /pkg/patches/0001-chore-musl-headers-compatibility-for-udev.patch
+        patch -p0 < /pkg/patches/musl.patch
+
+        meson setup build \
+          --buildtype=release \
+          -Dmode=release \
+          -Dlibdir=/usr/lib \
+          -Dversion-tag="{{ .systemd_version }}-talos" \
+          -Dselinux=enabled \
+          -Dblkid=enabled \
+          -Dkmod=enabled \
+          -Dopenssl=disabled \
+          -Dgshadow=false \
+          -Dutmp=false \
+          -Duserdb=false \
+          -Ddbus=disabled \
+          -Dglib=disabled \
+          -Dnss-mymachines=disabled \
+          -Dnss-myhostname=false \
+          -Dnss-resolve=disabled \
+          -Dnss-systemd=false \
+          -Dbacklight=false \
+          -Dbinfmt=false \
+          -Dbpf-framework=disabled \
+          -Dcoredump=false \
+          -Denvironment-d=false \
+          -Dfirstboot=false \
+          -Dhibernate=false \
+          -Dhostnamed=false \
+          -Dlink-udev-shared=false \
+          -Dlogind=false \
+          -Dmachined=false \
+          -Dnetworkd=false \
+          -Doomd=false \
+          -Dportabled=false \
+          -Dsysext=false \
+          -Danalyze=false \
+          -Dtimedated=false \
+          -Dtimesyncd=false \
+          -Dtmpfiles=false \
+          -Dpolkit=disabled \
+          -Dstatic-libudev=true \
+          -Dman=disabled \
+          -Dtests=false \
+          -Dwerror=false \
+          -Dsplit-bin=false \
+          -Dtranslations=false \
+          -Dxdg-autostart=false \
+          -Daudit=disabled \
+          -Dukify=disabled \
+          -Dkernel-install=false \
+          -Drfkill=false \
+          -Dvmspawn=disabled \
+          -Dhomed=disabled \
+          -Dremote=disabled \
+          -Dmountfsd=false \
+          -Dsysupdate=disabled \
+          -Dsysupdated=disabled \
+          -Dbootloader=enabled \
+          -Defi=true \
+          -Dsbat-distro-summary="Talos Linux" \
+          -Dsbat-distro-url=https://github.com/siderolabs/talos/issues \
+          -Dsbat-distro=talos
+    build:
+      - |
+        cd /tmp/systemd
```

Comment:
I guess easier to just copy what we need? or atleast only `/tmp/systemd/build` we won;t need sources at least

Reply:
Unsure, but I thought about keeping the full tree so that building something extra only requires adding options and a package to copy necessary parts

Reply:
since we don't push this stage should be fine :+1: